internal/tiltfile/starkit/fixture.go

```diff
@@ -12,25 +12,37 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"go.starlark.net/starlark"
+
+	"github.com/windmilleng/tilt/internal/testutils/tempdir"
 )
 
 // A fixture for test setup/teardown
 type Fixture struct {
-	tb         testing.TB
-	extensions []Extension
-	path       string
-	fs         map[string]string
-	out        *bytes.Buffer
-	useRealFS  bool // Use a real filesystem
+	tb               testing.TB
+	extensions       []Extension
+	path             string
+	temp             *tempdir.TempDirFixture
+	fs               map[string]string
+	out              *bytes.Buffer
+	useRealFS        bool // Use a real filesystem
```

Comment:
this is no longer used?

Reply:
It is still used.

<img width="957" alt="Screen Shot 2020-02-05 at 5 00 03 PM" src="https://user-images.githubusercontent.com/452453/73887155-fdb3aa00-4838-11ea-8727-ffbf1506d0d4.png">